pandas/tests/io/formats/test_format.py

```diff
@@ -2874,6 +2874,26 @@ def test_output_display_precision_trailing_zeroes(self):
             expected_output = "0     840\n1    4200\ndtype: float64"
             assert str(s) == expected_output
 
+    @pytest.mark.parametrize(
+        "value,expected",
+        [
+            ([9.4444], [9]),
+            ([0.49], [0.5]),
+            ([10.9999], [11]),
+            ([9.5444, 9.6], [10, 10]),
```

Comment:
The str result of `DataFrame(value)` should be hardcoded here as `expected` instead